### PR TITLE
add pr.js file

### PR DESCRIPTION
**Pull request by ZakharyanKaren**
TRY TO UNDERSTAND

### CODE REVIEW

**Comment by ZakharyanKaren** [comment]
gfdghfh

File: pr.js [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+  {
+    title: "Mr. Nobody",
+    author: "Catherine Steadman",
+  }
+);
+
+karenZakharyan.borrowBook(markTwainBook, readLoversLib);
+
+console.log(readLoversLib.checkReaderId(karenZakharyan.readerId));
+console.log(readLoversLib.lendBook(markTwainBook, karenZakharyan.readerId));
```
Comment (OfelyaMalkhasyan) [inline pr.js:261]: Everything is fine!

**Review by OfelyaMalkhasyan — COMMENTED** [review]
Ok

File: pr.js (hunk 1 above)
Comment (ZakharyanKaren) [reply pr.js:261]: Thank youuuuuu

**Pull request merged**